docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/03_Sites.md

```diff
@@ -28,7 +28,7 @@ command:
 INSERT INTO psc (Name, Alias, MRI_alias, Study_site) VALUES ('Montreal','MTL','MTL','Y');
 ```
 
-> Please refer to the [SQL Dictionary](SQL Dictionary.md) for additional information about
+> Please refer to the [SQL Dictionary](../../99%20-%20Developers/SQL_Dictionary.md) for additional information about
```

Comment:
Using the URL encodings for spaces is what the linked issue is trying to avoid. You'll want to change the path to `99_Developers` in order to resolve the issue with spaces in the filepaths. Every folder under `docs/wiki` will need to be updated.

Reply:
#6415